plezanje-net/web
# Image upload dialog

closes #229 

Adds option for uploading images on crag page and on route page.
Action to upload is inside the dropdown menu in the title row.

One that uploads an image is the contributor but not necessarily also the author of the image.
That is why a checkbox is present. The user can either claim that she is the author or uncheck this and enter the author manually (the real author might not be a pnet user).

FE now displays the author field bellow the image and not the contributor (user that uploaded) anymore.

We are not submitting image type anymore as we see not benefit in keeping track of this.

Also a changelog entry is added 'the old way'.

to test: 
Set environment var for the upload endpoint. !need to check this route on server before deploying!
upload some images to a crag and to a route.
test with: https://github.com/plezanje-net/api/pull/146
